examples/inventory_example.rb

```diff
@@ -26,7 +26,7 @@ def print_object(base, caption, indent = 0, recurse = true)
         puts "#{indentation}Ignoring #{rel} relationship due to ovirt error: #{err}"
       rescue RestClient::InternalServerError => err
         puts_caption(new_caption, indentation)
-        puts "#{indentation}Ignoring #{rel} relationship due to rest client error: #{err}"
+        puts "#{indentation}Ignoring #{rel} relationship due to rest client error: #{err.response}"
```

Comment:
Is response the same as `err.message`?  Can you clarify why and maybe link to [this change](https://github.com/marios/rest-client/pull/1/commits/a5b0998fdcdd366bf35aff29dc1299faaabb790e#diff-eb43f38b5b854e38aa016f90e0ce30baL107) to better explain why?